eventhubs/client/CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License
+Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For
+details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate
+the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to
+do this once across all repositories using our CLA.
+
+## Code of conduct
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact
+[opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Getting started
+
+Before working on a contribution, it would be beneficial to familiarize yourself with the process and guidelines used
+for the Azure SDKs so that your submission is consistent with the project standards and is ready to be accepted with
+fewer changes requested. In particular, it is recommended to review:
+
+- [Azure SDK README][github-general], to learn more about the overall project and processes used.
+- [Azure SDK Design Guidelines][design-guidelines], tounderstand the general guidelines for the SDKs across all
+  languages and platforms
+- [Azure SDK Design Guidelines for Java][java-spec], to understand the guidelines specific to the Azure SDKs for Java.
+
+## Development environment setup
+
+### Prerequisites
+
+- Java Development Kit (JDK) with version 8 or above
+- [Maven][maven]
+- Git
+- Microsoft Azure subscription
+    - You can create a free account at: https://azure.microsoft.com
+- Azure Event Hubs instance
+    - Step-by-step guide for [creating an Event Hub using the Azure Portal][event_hubs_create]
+
+## Building all the client libraries
+
+Open a command prompt/terminal:
+1. Execute: `git clone https://github.com/Azure/azure-sdk-for-java.git`
+1. Traverse to the repository root.
+1. Install the tooling and build the product by executing:
+    * `mvn install -Dinclude-non-shipping-modules -DskipTests -Dgpg.skip -f pom.client.xml`
```

Comment:
This doesn't build anything - it just does the install

Reply:
That's interesting. I always see it running tests, checkstyle, and stuff when I execute `mvn install`.
Is adding `mvn compile` prior to this better?
https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#Built-in_Lifecycle_Bindings

Reply:
Oh nevermind. it's becsause I always run `mvn clean install...`